From 33a5ec4977f36ce3a24c9ee824d9dd053b8cea04 Mon Sep 17 00:00:00 2001
From: Dima Pasechnik <dimpase@gmail.com>
Date: Fri, 8 May 2020 15:55:27 +0100
Subject: [PATCH 1/1] remove 1st and last file in .pc file

this causes problem if building with ksh, as they remain, causing a broken .pc file.
---
 fflas-ffpack.pc.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fflas-ffpack.pc.in b/fflas-ffpack.pc.in
index a2618d6..e34a744 100644
--- a/fflas-ffpack.pc.in
+++ b/fflas-ffpack.pc.in
@@ -1,4 +1,3 @@
-/------------------ fflas-ffpack.pc ------------------------
 prefix=@prefix@
 exec_prefix=@prefix@
 libdir=@prefix@/lib
@@ -11,4 +10,3 @@ Version: @VERSION@
 Requires: givaro >= 4.0.3
 Libs: @PARLIBS@ @PRECOMPILE_LIBS@ @BLAS_LIBS@
 Cflags: -I@includedir@  @BLAS_CFLAGS@  @PARFLAGS@ @PRECOMPILE_FLAGS@ @REQUIRED_FLAGS@
-\-------------------------------------------------------
\ No newline at end of file
-- 
2.26.2

